firebase-installations/src/main/java/com/google/firebase/installations/Utils.java

```diff
@@ -55,7 +60,7 @@ public long currentTimeInSecs() {
   /** Returns current time in milliseconds. */
   public long currentTimeInMillis() {
     // Mockito doesn't allow to mock static methods. As a result this util method is not static.
-    return System.currentTimeMillis();
+    return clock.millis(); // Returns current time in milliseconds.
```

Comment:
It's usually a bad sign if you have to add a comment to a simple method like this one.
How about you use the method name `#currentTimeMillis` since everybody will understand what it means?